regret/_api.py

```diff
@@ -107,7 +108,7 @@ def call_deprecated(*args, **kwargs):
             __doc__ = thing.__doc__
             if __doc__ is not None:
                 call_deprecated.__doc__ = self._new_docstring(
```

Comment:
So a "known" issue here for me which I didn't write up as a ticket is that `new_docstring` is likely too global -- I don't think we'll be able to have one single callable be passed to `_Deprecator` that can handle changing the docstring for any kind of deprecation (parameter, module, attribute, et al).

Just pointing that out in case / if it caused issue here when adding what you added.

Reply:
I will look at epytext support later :)